src/components/Table/TableBody/TableBodyRow/TableBodyRow.jsx

```diff
@@ -450,6 +456,7 @@ const TableBodyRow = ({
   ) : (
     <StyledTableRow
       key={id}
+      data-single-select={hasRowSelection}
```

Comment:
Wouldn't be simpler if we just propagate `hasRowSelection`?

```suggestion
      hasRowSelection={hasRowSelection}
```